File: python/The Full Counting Sort.py
limit = int(input())
ans = [""] * 11
for i in range(limit):
    x, y = input().split(" ")
    x = int(x)
    if i < limit / 2:
        ans[x] += "- "
    else:
        ans[x] += y + " "
print(*ans, sep="")

File: cpp/The Full Counting Sort.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

    int limit;
    cin >> limit;
    string ans[11] = { "" };
    for (int i = 0; i < limit; ++i) {
        int x;
        string y;
        cin >> x >> y;
        if (i < limit / 2)
            ans[x] += "- ";
        else
            ans[x] += y + " ";
    }

    for (string i : ans)
        cout << i;
    return 0;
}
